collector/stat_activity.go

```diff
@@ -157,5 +169,13 @@ func (c *statActivityScraper) Scrape(ctx context.Context, conn *pgx.Conn, versio
 	// postgres_stat_activity_oldest_snapshot_seconds
 	ch <- prometheus.MustNewConstMetric(c.snapshot, prometheus.GaugeValue, oldestSnapshot)
 
+	err = conn.QueryRow(ctx, statActivityScraperOldestSnapshotXidQuery).Scan(&oldestXmin)
+	if err != nil {
+		return err
+	}
+
+	// postgres_stat_activity_oldest_backend_xmin
+	ch <- prometheus.MustNewConstMetric(c.xmin, prometheus.GaugeValue, oldestXmin)
```

Comment:
I'm pretty sure now that this should be a counter, not a gauge (as long as my assertion about the query that retrieves this metric always holding a snapshot is correct)

Thoughts welcome though